### DISCUSSION TITLE
Option to duplicate team profiles: suggestion

### DISCUSSION THREAD

**Discussion by 01park** [PrestaShop/PrestaShop#40221] [Ideas]
I would like to have the option to duplicate a team profile in back office. 

This would speed things up :)

**Comment by khouloudbelguith** [comment]
Hi @01park,

Thanks for your report.
Ping @PrestaShop/prestashop-product-team what do you think?
![image](https://user-images.githubusercontent.com/16067358/68292911-2509a800-008d-11ea-87b2-76703d1232c2.png)

Thanks!

**Comment by TristanLDD** [comment]
Hi @01park, in order to better understand your need could you give us an example of a use case when you need to perform this action? 

Thanks!

**Comment by 01park** [comment]
The reason I came up with this suggestion is because lets say you have a group working on a project, but one person you want to give extra access. For this you will need to create a second profile with all the existing option. By duplicating an account, you will less likely make a mistake as you no longer need to select the options in a empty sheet. 

Example : Team SEO profile
duplicate this + more access for TEAM SEO manager

**Comment by hibatallahAouadni** [comment]
Ping @PrestaShop/product-team friendly reminder :wink:

**Comment by kpodemski** [comment]
Give it green light and I can handle this 👍
